src/reanimated2/platform-specific/webAnimationsData.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  BounceIn,
+  BounceInData,
+  BounceOut,
+  BounceOutData,
+} from './webAnimationsData/Bounce.web';
+import {
+  FadeIn,
+  FadeInData,
+  FadeOut,
+  FadeOutData,
+} from './webAnimationsData/Fade.web';
+import {
+  FlipIn,
+  FlipInData,
+  FlipOut,
+  FlipOutData,
+} from './webAnimationsData/Flip.web';
+import {
+  LightSpeedIn,
+  LightSpeedInData,
+  LightSpeedOut,
+  LightSpeedOutData,
+} from './webAnimationsData/Lightspeed.web';
+import { Pinwheel, PinwheelData } from './webAnimationsData/Pinwheel.web';
+import {
+  RollIn,
+  RollInData,
+  RollOut,
+  RollOutData,
+} from './webAnimationsData/Roll.web';
+import {
+  RotateIn,
+  RotateInData,
+  RotateOut,
+  RotateOutData,
+} from './webAnimationsData/Rotate.web';
+import {
+  SlideIn,
+  SlideInData,
+  SlideOut,
+  SlideOutData,
+} from './webAnimationsData/Slide.web';
+import {
+  StretchIn,
+  StretchInData,
+  StretchOut,
+  StretchOutData,
+} from './webAnimationsData/Stretch.web';
+import {
+  ZoomIn,
+  ZoomInData,
+  ZoomOut,
+  ZoomOutData,
+} from './webAnimationsData/Zoom.web';
+
+import type { AnimationData } from './animationParser';
+
+export const AnimationsData: Record<string, AnimationData> = {
+  ...FadeInData,
+  ...FadeOutData,
+  ...BounceInData,
+  ...BounceOutData,
+  ...FlipInData,
+  ...FlipOutData,
+  ...StretchInData,
+  ...StretchOutData,
+  ...ZoomInData,
+  ...ZoomOutData,
+  ...SlideInData,
+  ...SlideOutData,
+  ...LightSpeedInData,
+  ...LightSpeedOutData,
+  ...PinwheelData,
+  ...RotateInData,
+  ...RotateOutData,
+  ...RollInData,
+  ...RollOutData,
+};
+
+export const Animations = {
+  ...FadeIn,
+  ...FadeOut,
+  ...BounceIn,
+  ...BounceOut,
+  ...FlipIn,
+  ...FlipOut,
+  ...StretchIn,
+  ...StretchOut,
+  ...ZoomIn,
+  ...ZoomOut,
+  ...SlideIn,
+  ...SlideOut,
+  ...LightSpeedIn,
+  ...LightSpeedOut,
+  ...Pinwheel,
+  ...RotateIn,
+  ...RotateOut,
+  ...RollIn,
+  ...RollOut,
+};
+
+// Those are the easings that can be implemented using Bezier curves.
+// Others should be done as CSS animations
+export const WebEasings: Record<string, number[]> = {
```

Comment:
Why not
```suggestion
export const WebEasings = {
```
?

Reply:
Because without this type typescript says that string cannot be used to index `WebEasings` inside `getEasingFromConfig`,

Reply:
Changed a bit in [eb60eb6](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/eb60eb6f708f5a2385c72c3452412c25ba10b7bd)